<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>birthInfo</class>
 <widget class="QWidget" name="birthInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame
{
	background: #152034;
	border: 1px solid rgba(0,0,0,0.3);
	border-radius: 6px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_dateRollingBox" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="RollingBox" name="widget_year" native="true"/>
         </item>
         <item>
          <widget class="RollingBox" name="widget_month" native="true"/>
         </item>
         <item>
          <widget class="RollingBox" name="widget_day" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_bottom" native="true">
        <widget class="QLabel" name="label_age">
         <property name="geometry">
          <rect>
           <x>36</x>
           <y>6</y>
           <width>210</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgba(255,255,255,0.1);
border-radius: 6px;
font-size: 12px;
font-family: 微软雅黑;
color: #FFFFFF;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_astro">
         <property name="geometry">
          <rect>
           <x>288</x>
           <y>6</y>
           <width>210</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgba(255,255,255,0.1);
border-radius: 6px;
font-size: 12px;
font-family: 微软雅黑;
color: #FFFFFF;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>534</width>
           <height>14</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 10px;
font-family: 微软雅黑;
color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>出生日期不会出现在个人资料页哦～</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollingBox</class>
   <extends>QWidget</extends>
   <header location="global">rollingbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
